app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,49 @@
+@import "foundation_and_overrides";
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.footer {
+  background: transparent;
+  text-align: center;
```

Comment:
Properties should be ordered background, bottom, height, padding, position, text-align, width